v0x04/utils.py

```diff
@@ -1,20 +1,19 @@
-"""Utilities module for of_core OpenFlow v0x04 operations"""
-from kytos.core.events import KytosEvent
-from kytos.core.interface import Interface
-
-from napps.kytos.of_core.utils import emit_message_out
-
-from pyof.v0x04.symmetric.echo_request import EchoRequest
+"""Utilities module for of_core OpenFlow v0x04 operations."""
+from pyof.v0x04.common.action import ControllerMaxLen
 from pyof.v0x04.controller2switch.common import ConfigFlag, MultipartType
 from pyof.v0x04.controller2switch.multipart_request import (FlowStatsRequest,
                                                             MultipartRequest)
 from pyof.v0x04.controller2switch.set_config import SetConfig
-from pyof.v0x04.common.action import ControllerMaxLen
+from pyof.v0x04.symmetric.echo_request import EchoRequest
 from pyof.v0x04.symmetric.hello import Hello
 
+from kytos.core.events import KytosEvent
+from kytos.core.interface import Interface
+from napps.kytos.of_core.utils import emit_message_out
+
 
 def update_flow_list(controller, switch):
-    """Method responsible for request stats of flow to switches.
+    """Request stats of flow to switches.
```

Comment:
Please replace with the more correct phrasing "Request flow stats from switches."

Reply:
I will change here.